antarest/study/storage/abstract_storage_service.py

```diff
@@ -267,24 +269,25 @@ def import_output(
 
     def export_study(self, metadata: T, target: Path, outputs: bool = True) -> Path:
         """
-        Export and compress the study inside a ZIP file.
+        Export and compress the study inside a 7zip file.
 
         Args:
             metadata: Study metadata object.
             target: Path of the file to export to.
             outputs: Flag to indicate whether to include the output folder inside the exportation.
 
         Returns:
-            The ZIP file containing the study files compressed inside.
+            The 7zip file containing the study files compressed inside.
         """
         path_study = Path(metadata.path)
         with tempfile.TemporaryDirectory(dir=self.config.storage.tmp_dir) as tmpdir:
             logger.info(f"Exporting study {metadata.id} to temporary path {tmpdir}")
             tmp_study_path = Path(tmpdir) / "tmp_copy"
             self.export_study_flat(metadata, tmp_study_path, outputs)
             stopwatch = StopWatch()
-            zip_dir(tmp_study_path, target)
-            stopwatch.log_elapsed(lambda x: logger.info(f"Study {path_study} exported (zipped mode) in {x}s"))
+            with py7zr.SevenZipFile(target, "w") as szf:
+                szf.writeall(tmp_study_path, arcname=".")
```

Comment:
The parameter `arcname="."` creates an unwanted folder `.` at the root of the folder.